samples/application_development/event_manager/src/events/control_event.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "control_event.h"
+
+
+static void profile_control_event(struct log_event_buf *buf,
+				  const struct event_header *eh)
+{
+}
+
+EVENT_INFO_DEFINE(control_event,
+		  ENCODE(),
+		  ENCODE(),
+		  profile_control_event);
+
+EVENT_TYPE_DEFINE(control_event,
+		  true,
+		  NULL,
+		  &control_event_info);
```

Comment:
i don't like that event structure gets suffix and user must somehow know that after calling `EVENT_INFO_DEFINE(control_event,..)` it must use `control_event_info`. Imo, event info name should be explicit so that user does something like:
```
EVENT_INFO_DEFINE(control_event_info,
		  ENCODE(),
		  ENCODE(),
		  profile_control_event);

EVENT_TYPE_DEFINE(control_event,
		  true,
		  NULL,
		  &control_event_info);